### ISSUE TITLE
Merge behavior with imports

### ISSUE THREAD

**Issue by cmungall** [ontodev/robot#50]
After a merge, import directives remain - intentional?

Control this via a flag?

**Comment by jamesaoverton** [comment]
I think the import directives for merged imports should be removed, but I haven't thought through the details of how to accomplish this.

**Comment by cmungall** [comment]
From OGW:

```
	public void mergeImportClosure(boolean isRemovedImportsDeclarations) throws OWLOntologyCreationException {
		OWLOntologyManager manager = getManager();
		Set<OWLOntology> imports = sourceOntology.getImportsClosure();
		for (OWLOntology o : imports) {
			if (o.equals(sourceOntology))
				continue;
			
			String comment = "Includes "+o;
			LOG.info(comment);
			addCommentToOntology(sourceOntology, comment);
			
			manager.addAxioms(sourceOntology, o.getAxioms());
		}
		Set<OWLImportsDeclaration> oids = sourceOntology.getImportsDeclarations();
		for (OWLImportsDeclaration oid : oids) {
			RemoveImport ri = new RemoveImport(sourceOntology, oid);
			getManager().applyChange(ri);
		}
	}
```

Might be useful to retain the provenance optionally

**Comment by jamesaoverton** [comment]
I ran into this problem again, tried a few things, and was frustrated with the results.

1. The example code above works fine for removing all `owl:imports` declarations.

2. `MergeOperation.mergeInto` takes a list of ontologies that you want to merge into the target ontology. Usually that's all of the imported ontologies, but not always. I tried to be clever and just remove imports for the list of ontologies to merge, using OBI as my test case. But sometimes OBI imports by ontology IRI, and sometimes by version IRI, so I had to remove both. Even then, sometimes the IRI in the import declaration is neither the ontology IRI nor the version IRI. Examples:

```
<owl:imports rdf:resource="http://purl.obolibrary.org/obo/bfo/2014-05-03/classes-only.owl"/>
<owl:imports rdf:resource="http://purl.obolibrary.org/obo/ro/releases/2015-10-07/core.owl"/>
```

If we can't automatically remove the specified imports in a reliable way, then I guess the best option is an explicit option such as `--collapse-import-closure` in #92.
